package.json

```diff
@@ -96,6 +96,7 @@
     "@storybook/addons": "^5.2.4",
     "@storybook/react": "^5.2.4",
     "@storybook/theming": "^5.2.4",
+    "@testing-library/react": "^9.3.2",
```

Comment:
Could we add this in a separate PR instead? 😅

Reply:
I could... I'll be writing unit tests in this branch with this package, so I'm not sure if that additional step is needed.

Reply:
It just seems like a big deal... to introduce this new testing library & approach. This work isn't tied to an issue, and it is slipped into this PR about something different. So tracking the introduction of it in release notes will be difficult when this is squashed + merged, plus again there's no issue/work tied to it.

Reply:
Thanks - I added this in https://github.com/carbon-design-system/ibm-security/pull/245 as I ended up leveraging it a number of other PRs.

I don't think this will be tracked in release notes as it's build-related, but I agree it's good practice to have the commit in the history if for some reason we need to return to the change.